app/views/api_docs/pages/home.md

```diff
@@ -0,0 +1,163 @@
+---
+page_title: Register API documentation
+title: Register API documentation
+---
+
+This is API documentation for the Department for Education (DfE)’s Register
+trainee teachers service.
+
+Register has replaced the HESA service for teacher trainees. All vendors of
+student record systems (SRS) and some training providers will need to make
+changes to integrate with Register.
+
+# What this API is for
+
+The Register API allows providers to import trainee records from their student
+record systems and to keep those records synchronised as they are modified.
+
+Providers can use the API for:
+
+- Looking up a trainee record in Register and returning details about that trainee.
+- Retrieving a list of trainee records in Register for a given academic year.
+- Creating new trainee records.
+- Updating individual trainee records.
+- Withdrawing and deferring a trainee.
+- Looking up reference data such as country code, school URNs etc.
+
+# What the API doesn't support
+
+
+# How do I connect to this API?
+
+## Authentication and authorisation
+
+Requests to the API must be accompanied by an authentication token.
+
+Each token is associated with a single provider. It will grant access to
+trainee records registered by that provider. You can get a token by writing to
+becomingateacher@digital.education.gov.uk.
+
+For instructions on how to authenticate see the <API reference>.
+
+# Design principles
+
+## Trainee identifiers
+Trainees are identified in the API by an internally generated unique `id`
+value. The API uses the `id` to specify a particular trainee record. For
+example, the API endpoint that updates a single trainee record requires that
+`id` value as part of it's URL.
+
+For example, the following request updates the trainee record with `id` value
+`gQGecNYwLCqPMTgzW7N2bURi`.
```

Comment:
I'm deliberately calling the trainee `slug` attribute `id` here because if we are going to use this as the unique identifier we should probably avoid exposing the 'real' `id` attribute.